builders/tanssi-network/.pages

```diff
@@ -1,7 +1,7 @@
-title: Tanssi Network
+title: Tanssi Networks
 hide: false
 nav:
   - index.md
-  - networks
-  - 'Endpoints': 'endpoints.md'
-  
+  - 'Dancebox': 'dancebox.md'
```

Comment:
I'm not super conviced by this tbh.

What about having:

- TestNet
   |- Dancebox
   |- Demo EVM Appchain

Reply:
Yeah, I'm totally cool with that. I just didn't love "Overview" for the Dancebox page.

So, then when we add Tanssi, the structure will be?:

- TestNet
  - Dancebox
  - Demo EVM Appchain
- Tanssi MainNet

Reply:
In this section, we have the "Network Endpoints" article which doesn't add anything of value, why keep it?

Reply:
Yeah, I debated deleting it, but the reason why I kept it is the same reason we have a similar setup on Moonbeam, and that is for visibility.

I'm on the fence on this TBH. So, if the general consensus is that it doesn't make sense to keep it, let's remove it

Reply:
I think having a specific Network Endpoint section is helpful because that is something developers generally look for to get started.

Maybe if we feel that each network page is not necessary because it brings very little value we can just have a Network Endpoint section and that is it

Reply:
On the Moonbeam docs, we see that the Get Started with Moon* pages actually get more traffic than the Network Endpoints page, but overall all of these pages get a high number of visitors and high number of events, meaning that users are interacting with that page, probably copying what they need, and then quickly leaving the page. So from what I can tell, both pages seem to allow users quickly get what they need. 

I know Moonbeam is different, but I think we should just keep both for now and see how they perform.